src/shared_context.h

```diff
@@ -219,6 +222,6 @@ extern uint32_t eth2WithdrawalIndex;
 #endif
 
 void reset_app_context(void);
-const uint8_t *parseBip32(const uint8_t *, uint16_t *, bip32_path_t *);
+const uint8_t *parseBip32(const uint8_t *, uint8_t *, bip32_path_t *);
```

Comment:
I think having parameters names in the prototype helps understand the function

Reply:
Fixed in [4db1e99](https://github.com/LedgerHQ/app-ethereum/pull/327/commits/4db1e99fabbae80aa4288b035f4a012d5c6273f2)